### PR TITLE
Debug player

### LINKED ISSUE #63: Debug mode without restrictions
Add a debug mode so one can see and execute locked missions
Maybe a toggleable UI control.
Or maybe just disable the lock on the debug version

### PR DESCRIPTION
**Pull request by privat**
Simplify debugging:
- autolog with a player (no need to login to test)
- show and allow to play locked mission (by the way, the lock was done client-side. this could be better)

close #63

### CODE REVIEW

**Comment by ppepos** [comment]
> by the way, the lock was done client-side. this could be better

lol nice find O_o

**Comment by ppepos** [comment]
+1

**Comment by Morriar** [comment]
> > by the way, the lock was done client-side. this could be better
> 
> lol nice find O_o

Except your session, nothing is locked server side for now.

File: www/directives/player/mission.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 			<p class='lead text-danger'>Mission locked</p>
 		</div>
 	</div>
-	<div ng-if='missionCtrl.missionStatus.status != "locked"'>
+	<div ng-if='missionCtrl.missionStatus.status != "locked" || true'>
```
Comment (Morriar) [inline www/directives/player/mission.html]: Are you sure?

Comment (privat) [reply www/directives/player/mission.html]: This to denote the temporary workaround of the PR. I can just remove the `ng-if` if you prefer

Comment (Morriar) [reply www/directives/player/mission.html]: If you remove the if, reopen the issue on locked missions.

**Comment by privat** [comment]
reroll: removed `|| true`

**Pull request merged**